contracts/ActivePool.sol

```diff
@@ -80,6 +86,20 @@ contract ActivePool is
 		renounceOwnership();
 	}
 
+	function setStakingAdminAddress(address _stakingAdminAddress) external {
+		require(!isStakingAdminInitialized, "ActivePool: staking admin already initialized");
```

Comment:
`require(stakingAdmin == address(0))`

Then, you can remove the isStakingAdminInitialized